src/test/java/io/confluent/idesidecar/restapi/resources/ConnectionsResourceTest.java

```diff
@@ -469,60 +477,99 @@ void listConnections_returnsAllConnections() {
     assertConnectionList(expectedList, actualList);
   }
 
-  @ParameterizedTest
-  @NullAndEmptySource
-  void updateConnectionWithNullOrEmptyIdShouldFail(String id) {
-    var originalId = "c1";
-    var connectionSpec = ccloudTestUtil.createConnection(
-        originalId, "Connection 1", ConnectionType.LOCAL);
+  @ParameterizedTest(name = "{0}")
+  @MethodSource("partialPayloadParams")
+  void updateConnectionViaPatchWithPartialPayloadsShouldSucceed(
+      String testName,
```

Comment:
Sorry, missed this in my earlier review: Let's keep the PUT tests as is and create new ones for PATCH.

Reply:
eek! didn't realize I removed this. thanks for catching